sklearn/feature_selection/tests/test_mutual_info.py

```diff
@@ -47,15 +47,15 @@ def test_compute_mi_cc(global_dtype):
     I_theory = np.log(sigma_1) + np.log(sigma_2) - 0.5 * np.log(np.linalg.det(cov))
 
     rng = check_random_state(0)
-    Z = rng.multivariate_normal(mean, cov, size=1000).astype(global_dtype)
+    Z = rng.multivariate_normal(mean, cov, size=1000).astype(global_dtype, copy=False)
 
     x, y = Z[:, 0], Z[:, 1]
 
     # Theory and computed values won't be very close, assert that the
     # first figures after decimal point match.
     for n_neighbors in [3, 5, 7]:
         I_computed = _compute_mi(x, y, False, False, n_neighbors)
-        assert_allclose(I_computed, I_theory, 1)
+        assert_allclose(I_computed, I_theory, rtol=1e-1)
```

Comment:
we should either use atol or rewrite the comment because ``first figures after decimal`` means absolute tolerance

Reply:
Indeed, I did changes here brainlessly.